owncloudApp/build.gradle

```
@@ -115,12 +115,22 @@ android {
     buildTypes {
 
         release {
+            resValue "string", "app_name_buildtype", "@string/app_name"
+            resValue "string", "dynamic_authority", "@string/authority"
+            resValue "string", "dynamic_search_suggest_authority", "@string/search_suggest_authority"
+            resValue "string", "dynamic_document_provider_authority",  "@string/document_provider_authority"
+            resValue "string", "dynamic_file_provider_authority",  "@string/file_provider_authority"
             if (System.env.OC_RELEASE_KEYSTORE) {
                 signingConfig signingConfigs.release
             }
         }
 
         debug {
+            resValue "string", "app_name_buildtype", "Debug Owncloud"
```

```suggestion
            resValue "string", "app_name_buildtype", "Debug ownCloud"
```

Maybe already fixed, but to be sure, ownCloud is lowercase "o" and uppercase "C"

---

Thanks for pointing to it !